build.gradle

```diff
@@ -458,3 +458,23 @@ task createWebsite(dependsOn: ['copyPDF', 'copyWebsiteContent', 'copyCompiledDoc
 task createDocs(dependsOn: createWebsite)
 
 task createDocsNoTest(dependsOn: ['copyPDF', 'copyWebsiteContent', 'copyCompiledDocs', 'copyHailDocsNoTest', 'copyPythonDirToTmp'])
+
+task assemblePackage(type: Copy, dependsOn: ['createDocs' , 'shadowJar']) {
+    from('python') {
+        into 'python'
+    }
```

Comment:
Not really a comment on the PR, but generally when we create the python zip we throw everything together (docs, sphinx config files, tests, etc.) that probably don't belong in the distribution.  I don't know much about python packing, so just an unhelpful observation.